docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This dependency visibility ensures that the build system can accurately collect, inquire, analyze, and manage all dependencies during the build process.
+
+In {ProductName}, hermetic builds are achieved by blocking network access to the build process. This means that all the dependencies must be pre-fetched before the build can start.
+
+IMPORTANT: If you do not vendor all the required dependencies in a git repository before blocking network access, your build may fail. 
+
+Hermetic builds also focus on establishing a build pipeline that captures and maintains a precise record of the build's origin, dependencies, and modifications. For more information on the importance of provenance, see link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/concepts/slsa/con_slsa-conformity/[Supply chain security through SLSA conformity].
+
+
+.Prerequisites
+
+* You have an link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_upgrade_build_pipeline/[upgraded build pipeline].
```

Comment:
Do you want to replicate this link in the requirements on the Cach2 prefetch page as well?